packages/next/build/webpack/plugins/profiling-plugin.ts

```diff
@@ -99,7 +99,32 @@ export class ProfilingPlugin {
           return
         }
 
-        const span = trace('build-module', compilerSpan.id)
+        const moduleType = (() => {
+          if (!module.userRequest) {
+            return ''
+          }
+
+          if (module.userRequest.endsWith('.css')) {
```

Comment:
jsx is missing. Do we need to support all extensions from next config `pageExtensions`? If I add custom extension like `.mjs` it will miss moduleType

maybe merge all module.userRequest.endsWith calls?
```js
extensions = ['ts', 'tsx', ...]
return extensions.find(ext => module.userRequest.endsWith('.' + ext))
```

not blocking